.changes/unreleased/Fixes-20240610-195300.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Use model alias for the CTE identifier generated during ephemeral materialization
+time: 2024-06-10T19:53:00.086488231Z
+custom:
+    Author: jeancochrane
+    Issue: "5273"
```

Comment:
This is the issue number in dbt-core; I wasn't sure whether I should make another one in this repo as well. Happy to do so if that's the correct procedure.

Reply:
This works for now!